docs/README.md

```diff
@@ -65,32 +75,17 @@ using the non-extended version:
 error: failed to transform resource: TOCSS: failed to transform "sass/style.sass" (text/x-sass): this feature is not available in your current Hugo version
 ```
 
-#### Installing static assets
-
-The OPA website requires some static assets installable via npm:
-
-```bash
-npm install
-```
-
-#### Serving the site
+#### Serving the full site
 
 From this directory:
 
 ```shell
 make serve
 ```
 
-Open your browser to http://localhost:1313 to see the site running locally. The docs
-are available at http://localhost:1313/docs.
+Watch the console output for a localhost URL (the port is randomized). The docs are
```

Comment:
The port is only randomized if Hugo is already running, otherwise it defaults to 1313.

Reply:
For hugo yea, but netlify in front of hugo takes a random port. This PR changes `make serve` to spin up netlify locally.

Reply:
Ah, that makes sense. Haven't used the new CLI, so that's good to know.